lib/matplotlib/animation.py

```diff
@@ -686,28 +688,35 @@ def save(self, filename, writer=None, fps=None, dpi=None, codec=None,
 
         *fps* is the frames per second in the movie. Defaults to None,
         which will use the animation's specified interval to set the frames
-        per second.
+        per second. This argument is only valid if *writer* is not a
+        :class:`MovieWriter` instance.
```

Comment:
Rather than be repetitive, I would segregate the documentation for these parameters and preface them with a statement explaining that they aren't meant to be used directly in this class, but for the subclasses (assuming I am understanding this right).